events/masterPanels.js

```diff
@@ -59,7 +61,21 @@ module.exports = (client) => {
             })
 
             let r = await interaction.reply({ embeds: [embeds[0]], fetchReply: true })
-            client.buttonPaginator(interaction.user.id, r, embeds, 1)
+            client.buttonPaginator(interaction.user.id, r, embeds, 1, { deleteOnEnd: true })
+        }
+        if (interaction.customId == "dev-maintenance") {
+            let maintenance = db.get("maintenance")
+            if (maintenance) {
+                db.delete("maintenance")
+                await interaction.reply({ content: "Maintenance mode disabled", ephemeral: true })
+                let commit = require("child_process").execSync("git rev-parse --short HEAD").toString().trim()
+                let branch = require("child_process").execSync("git rev-parse --abbrev-ref HEAD").toString().trim()
+                client.user.setPresence({ activities: [{ name: client.user.username.toLowerCase().includes("beta") ? "testes gae on branch " + branch + " and commit " + commit : "Wolvesville Simulation!", type: "PLAYING" }], status: "online" })
```

Comment:
"testes gae on branch" is the beta bot's status for devs to see what branch they are on and what commit. It has been like this since I joined the dev team